File: week-02/day-04/src/PrintBigger.java
import java.util.Scanner;

public class PrintBigger {
    // Write a program that asks for two numbers and prints the bigger one
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Give me two numbers:");

        int number1 = sc.nextInt();
        System.out.println(number1);

        int number2 = sc.nextInt();
        System.out.println(number2);

        printBigger(number1, number2);
    }
    public static void printBigger(int number1, int number2) {
        if (number1 > number2) {
            System.out.println(number1 + " is the bigger number.");
        } else if (number2 > number1) {
            System.out.println(number2 + " is the bigger number.");
        } else {
            System.out.println("Neither one is bigger.");
        }
    }
}

File: PythonPractice/1211/PrintBigger.py
# Write a program that asks for two numbers and prints the bigger one


number1 = int(input("Give me the first number: "))
number2 = int(input("Give me the second number"))


def printbigger(number1, number2):
    if number1 > number2:
        print(number1)
    elif number2 > number1:
        print(number2)
    else:
        print("The numbers are equal")


printbigger(number1, number2)


